session/session.go

```diff
@@ -189,11 +190,11 @@ func (s *session) handleRestoredCh(pch *pclient.Channel) {
 	if pch.Phase() != pchannel.Acting {
 		return
 	}
-	peers := pch.Peers()
-	parts := make([]perun.Peer, len(peers))
-	aliases := make([]string, len(peers))
+	peerIDs := pch.Peers()
+	parts := make([]perun.PeerID, len(peerIDs))
+	aliases := make([]string, len(peerIDs))
 	for i := range pch.Peers() {
-		p, ok := s.contacts.ReadByOffChainAddr(peers[i])
+		p, ok := s.idProvider.ReadByOffChainAddr(peerIDs[i])
```

Comment:
```suggestion
		p, ok := s.idProvider.ReadByOffChainAddr(partOffChainAddrs[i])
```
In accordance with the variable rename in the above comment.